src/ui/components/todo-item/component.ts

```diff
@@ -4,27 +4,36 @@ const ENTER_KEY = 13;
 const ESCAPE_KEY = 27
 
 export default class TodoItem extends Component {
-  @tracked editing = false;
-  @tracked newTitle;
+  @tracked editing: boolean = false;
+  @tracked newTitle: string;
 
   beginEdit() {
     this.editing = true;
     this.newTitle = this.args.todo.title;
+    
+    // temporary solution
+    setTimeout(() => {
```

Comment:
Unfortunately I couldn't find better solution for now and I'm willing to discuss this matter.